src/components/HURUmap/Tutorial/TutorialItem/index.js

```diff
@@ -0,0 +1,93 @@
+import RichTypography from "@commons-ui/core/RichTypography";
+import { Grid, Typography, IconButton } from "@material-ui/core";
+import { useTour } from "@reactour/tour";
+import Image from "next/image";
+import PropTypes from "prop-types";
+import React from "react";
+
+import TutorialIcon from "./TutorialIcon";
+import useStyles from "./useStyles";
+
+import { ReactComponent as CloseIcon } from "@/pesayetu/assets/icons/Component 108 – 5.svg";
+
+function TutorialItem({
+  activeStep,
+  description,
+  title,
+  image,
+  selector,
+  ...props
+}) {
+  const classes = useStyles(props);
+  const { setIsOpen } = useTour();
+  const handleClose = () => {
+    setIsOpen(false);
+  };
+
+  return (
+    <div className={classes.root}>
+      <Grid
+        className={classes.header}
+        container
+        justifyContent="space-between"
+        alignItems="center"
+      >
+        <Grid item>
+          <Typography
+            id={`tutorial-title-${activeStep - 1}`}
```

Comment:
What is the point of passing in `index + 1` as `activeStep` to only use `activeStep - 1` here?